### PR TITLE
Fix: Auto lock can not run correct

### PR DESCRIPTION
**Pull request by hoailinh1210**
### Summary (Please recap what you changed or fixed)
- Root cause:
       - `browser` undefined
- Solution: 
       - Retry to load browser with interval
### Checklist (If you won't pass this checklist please comment why)

-   [x] I removed all commented or unnecessary code.
-   [ ] Provide the screenshots due to UI changed or bug fixed
-   [ ] My code has covered these test cases (please list down your tested cases):

### CODE REVIEW

File: client/src/components/hooks/useLockAccountWhenIdle.js [hunk 1]
```diff
@@ -19,30 +21,36 @@ const useLockAccountWhenIdle = () => {
...
 
-		try {
-			if (!browser || isUsingLedger) {
-				return;
-			}
-
-			browser.runtime.onMessage.addListener((event) => {
-				// Make sure login modal does not open.
-				if (typeof event !== 'object') {
+		let retryTime = 0;
```
Comment (duc-q) [inline client/src/components/hooks/useLockAccountWhenIdle.js:24]: Please make sure `retryTime` is updated

Comment (hoailinh1210) [reply client/src/components/hooks/useLockAccountWhenIdle.js:24]: just updated it, thanks bro 🥇

File: client/src/apiServices/friendlyMarket/moduleBytes/index.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
+
```
Comment (djhahe) [inline client/src/apiServices/friendlyMarket/moduleBytes/index.js:2]: this file should not in this PR

**Review by duc-q — APPROVED** [review]
LGTM

File: client/src/apiServices/friendlyMarket/moduleBytes/index.js (hunk 2 above)
Comment (hoailinh1210) [reply client/src/apiServices/friendlyMarket/moduleBytes/index.js]: Removed this, thanks bro

**Pull request merged**